lib/pf/web/constants.pm

```diff
@@ -93,18 +93,15 @@ Readonly::Scalar our $URL_GAMING_REGISTRATION   => '/gaming-registration';
 Readonly::Scalar our $URL_DEVICE_REGISTRATION   => '/device-registration';
 Readonly::Scalar our $URL_DEVICE_REG_LOGOUT     => '/device-registration/logout';
 
-# External Captive Portal detection constant
-Readonly::Scalar our $REQ_CISCO                 => 'ap_mac';
-Readonly::Scalar our $REQ_MAC                   => 'mac';
-Readonly::Scalar our $REQ_ARUBA                 => 'apname';
-Readonly::Scalar our $REQ_CISCO_PORTAL          => '/cep(.*)';
```

Comment:
After verification, the "cep" part was never handled by the user / configuration parameter itself but was added in the code based of if $sessionid was part of the URL.
There is then, no check to do.
I also reworked the logic of the session id no longer needed to be in the configuration since it is a de facto flow with Cisco (https://github.com/inverse-inc/packetfence/pull/1613/commits/d842faa8fdc029e815f66337d55b5831a52940d8)